Demo/TableViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+class TableCell: UITableViewCell {
```

Comment:
Should we rename `TableViewController` to `TableController` instead? Or rename to `TableViewCell`